test/e2e/e2e_test.go

```diff
@@ -155,11 +155,16 @@ func checkConnectivityPingToHost(f *framework.Framework, nodeName, podName, host
 		_, ok := pod.Annotations[podNetworkAnnotation]
 		return ok, nil
 	})
+	if err != nil {
+		// FIXME: wait.PollImmediate fails to check the pod annotations
```

Comment:
Sorry, I was thinking the fix in this commit applied to other tests, not just the broken exgw ones. I guess this commit should wait until the commit that fixes those tests rather than merging now.

otherwise the PR lgtm